openwisp_monitoring/device/base/models.py

```diff
@@ -398,6 +398,19 @@ def get_active_metrics(cls):
             cls._active_metrics = active_metrics
         return cls._active_metrics
 
+    @classmethod
+    def get_critical_checks(cls):
```

Comment:
We can easily loop over the `CRITICAL_DEVICE_METRICS` list in the device/settings.py. 

I chose to add a class method, to make the methods more consistent. Now, we have two classmethod for the DeviceMonitoring model: 

- DeviceMonitoring.get_active_metrics()
- DeviceMonitoring.get_critical_checks()


What do you think @nemesifier? Shall we keep this or move the logic to app settings?

Reply:
Looks good :+1: